packages/@aws-cdk/aws-apigateway/lib/authorizers/lambda.ts

```diff
@@ -15,23 +15,31 @@ export interface TokenAuthorizerProps {
    *
    * @default - CDK will use the uniqueId assigned to this construct.
    */
-  readonly name?: string;
+  readonly authorizerName?: string;
 
   /**
-   * The authorizer lambda function.
+   * The handler for the authorizer lambda function.
+   *
+   * The handler must follow a very specific protocol on the input it receives and the output it needs to produce.
+   * API Gateway has documented the handler's input specification
+   * {@link https://docs.aws.amazon.com/apigateway/latest/developerguide/api-gateway-lambda-authorizer-input.html | here} and output specification
+   * {@link https://docs.aws.amazon.com/apigateway/latest/developerguide/api-gateway-lambda-authorizer-output.html | here}.
    */
-  readonly function: lambda.IFunction;
+  readonly handler: lambda.IFunction;
 
   /**
-   * The name of the header in the request that contains the authorization token as submitted by the client.
+   * The request header mapping expression for the bearer token. This is typically passed as part of the header, in which case
+   * this should be `method.request.header.AuthToken` where AuthToken is the header containing the bearer token.
+   * @see https://docs.aws.amazon.com/apigateway/api-reference/link-relation/authorizer-create/#identitySource
+   * @default 'method.request.header.auth_token'
```

Comment:
Is that a common default? I thought `Bearer` would make more sense but I might be mistaking.

Reply:
A search on the internet didn't yield 'Bearer', although that's what they're called.

Google & Github use `Authorization`, while Twitter and some others use `auth_token`.

Reply:
https://swagger.io/docs/specification/authentication/bearer-authentication/

Reply:
So probably `Authorization`